docs/ENVIRONMENT.md

```diff
@@ -41,6 +41,9 @@ All tools:
   debugging, in order to test generators and other code against specific kernel
   command lines.
 
+* `$SYSTEMD_CRYPTTAB` — if set, use this path instead of /etc/crypttab. Only
+  useful for debugging. Currently only supported by systemd-cryptsetup-generator.
```

Comment:
sounds like something to have for /etc/fstab too...

Reply:
OK, will add.

Reply:
(doesn't have to be this PR, I meant in general)